src/Assignments/Monsters/students/11th/Kangyejiii/Monsters.js

```diff
@@ -30,15 +30,38 @@ class Monsters extends Component {
 
   // SearchBox 에 props로 넘겨줄 handleChange 메소드 정의
 
+  componentDidMount() {
+    fetch("https://jsonplaceholder.typicode.com/users")
+    .then(res=>res.json())
+    .then(res=>this.setState({monsters : res}));
+  }
+
+  handleChange = (e) => {
+    this.setState({userInput: e.target.value});
+    }
```

Comment:
여기도 아래처럼 closing bracket 위치 변경해주세요!
```suggestion
  handleChange = (e) => {
    this.setState({userInput: e.target.value});
  }
```